code/modules/shuttle/supply.dm

```diff
@@ -114,18 +114,25 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		setupExports()
 
 	var/msg = ""
+	var/matched_bounty = FALSE
 	var/sold_atoms = ""
 
 	for(var/place in shuttle_areas)
 		var/area/shuttle/shuttle_area = place
 		for(var/atom/movable/AM in shuttle_area)
-			if(AM.anchored || iscameramob(AM))
```

Comment:
This prevents ghosts, dviewmob, effects etc from being sold, you'll need to replace it with something and not just assume it does nothing

Reply:
I was wondering about that, but can you take another look? The two checks are still there, but have been moved around a bit.

Reply:
Still applies to things in bounties that shouldn't move. I'd rather this wasn't leaky so you don't have to worry about object properties when adding new ones.

Reply:
Mechs and a few other bounty items are anchored though. I need to know the right way to handle it so that anchored items can still be bountied.

When mech exports were originally added a snowflake `istype(AM, /obj/mecha)` check was put in to make it work. But then this check got removed in #29049 which left mech exports broken for about a year.